wsat-simple/src/main/webapp/WEB-INF/jboss-web.xml

```diff
@@ -15,7 +15,12 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-<jboss-web>
+<!DOCTYPE jboss-web>
+<jboss-web xmlns="http://www.jboss.com/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xsi:schemaLocation="http://www.jboss.org/schema/jbossas 
+    http://www.jboss.org/schema/jbossas/jboss-web_7_2.xsd">
+    <!-- Configure to enable WebSockets -->
     <context-root>/</context-root>
```

Comment:
Does this QS work with Web Sockets? I don't think we (the transactions team) have ever tried WS-AT or WS-BA with Web Sockets.